### ISSUE TITLE
Some CSS spruce up and additional sheet if you're interested

### ISSUE THREAD

**Issue by RandallBurt** [GrumpyOldDM/STA_DemoSheet#1]
So, I was really impressed and excited to see your sheet and started playing around with it (hence my comments on your PR to Roll20). Its really great work and I don't want to step on your toes, but I spent a couple days playing with the CSS and layout and was wondering if you'd be interested in a PR from me to add to your PR to Roll20? I also went and added the Starship Registry Entry sheet as well (I changed the header to a dropdown that changes the sheet configuration). Again, I don't want to step on toes here and was really excited when you submitted the sheet and I had just gotten the core PDF and wanted to try and match the sheets there as close as possible.

![examplesheet](https://user-images.githubusercontent.com/29293921/28054580-0b7964e2-65db-11e7-858d-fbcc72b7324e.jpg)
![examplestarshipsheet](https://user-images.githubusercontent.com/29293921/28054583-0fb23642-65db-11e7-837c-39cc1516ed55.jpg)

**Comment by RandallBurt** [comment]
Doh! I just realized I raised this on the wrong repository. Let me know if you want me to move it @GrumpyOldDM .

**Comment by GrumpyOldDM** [comment]
I'm open indeed, I was just putting the finishing touches on the ship sheet over here, do you want to collaborate together on the single sheet? (does Roll20 even allow that?)

**Comment by RandallBurt** [comment]
I don't think roll20 allows it but I can send a PR to your repo and you can copy/pasta into a session that we both are part of. R20 doesn't allow two people to change the game settings, just the game owner. I'm doing day job things but could send you a PR later tonight. Should I PR this repo or your Roll20 fork?

**Comment by GrumpyOldDM** [comment]
I just did the finishing touches on the Starship sheet and moved over to worker scripts vs auto calculating fields - feel free to fork from here as this is my working copy. Once you make your changes I'll incorperate into the master Roll20  and get you added in the json as an author as well

**Comment by RandallBurt** [comment]
Hey I haven't forgotten about this. I've needed to refactor a bit of the
layout to get things working and was stumpped on getting the dynamic
shield/stress checkboxes for a bit. Should be sorted now, but I needed to
change some attribute names to match conventions for working with tokens
and whatnot (_max instead of _full etc.) so I need to verify all the
attribute names again and probably write a conversion script. I can send
you wip pr tomorrow or saturday tho.
